go-console/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.9"
+services:
+  __container__-debug:
+    build:
+      context: .
+      dockerfile: Dockerfile.debug
+    image: ${LOCAL_REGISTRY}:5002/__container__-debug:${TAG}
+    ports:
+      - 2345:2345
+      - 2299:2299
```

Comment:
The port `2299` is already used by `rust-console` application. Please if possible use another one that is not listed on the https://github.com/toradex/vscode-torizon-templates/blob/main/templates.json